Sessions/2e-Dictionaries-and-json.ipynb

```diff
@@ -1155,43 +1082,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": null,
    "id": "7ae1c222",
    "metadata": {
     "slideshow": {
      "slide_type": "fragment"
     }
    },
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "Counter({'Alice': 1,\n",
-       "         'was': 1,\n",
-       "         'beginning': 1,\n",
-       "         'to': 2,\n",
-       "         'get': 1,\n",
-       "         'very': 1,\n",
-       "         'tired': 1,\n",
-       "         'of': 2,\n",
-       "         'sitting': 1,\n",
-       "         'by': 1,\n",
-       "         'her': 1,\n",
-       "         'sister': 1,\n",
-       "         'on': 1,\n",
-       "         'the': 1,\n",
-       "         'bank': 1,\n",
-       "         'and': 1,\n",
-       "         'having': 1,\n",
-       "         'nothing': 1,\n",
-       "         'do': 1})"
-      ]
-     },
-     "execution_count": 20,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "word_counts = Counter(sentence)\n",
```

Comment:
I don't know if this is intended, but `sentence` here is the string: so you are getting a counter of characters in the string, not a counter of tokens. This should be `word_counts = Counter(word_list)`, right?

Reply:
Yes, correct! Thanks for spotting this ;-) Would make a good exercise, hehe